spec.bs

```diff
@@ -3147,14 +3235,70 @@ interface InterestGroupScriptRunnerGlobalScope {
 
 </pre>
 
+<pre class="idl">
+[Exposed=InterestGroupBiddingAndScoringScriptRunnerGlobalScope]
+interface ForDebuggingOnly {
+  undefined reportAdAuctionWin(DOMString url);
+  undefined reportAdAuctionLoss(DOMString url);
+};
+
+[Exposed=InterestGroupBiddingAndScoringScriptRunnerGlobalScope]

```

Comment:
I think you'll need `Global=InterestGroupBiddingAndScoringScriptRunnerGlobalScope` here, in order to use `Exposed=InterestGroupBiddingAndScoringScriptRunnerGlobalScope` above. See https://webidl.spec.whatwg.org/#Global.